systemd/unit.go

```diff
@@ -57,7 +57,7 @@ func (p *P) statsToPod(stats map[string]*unit.State) *corev1.Pod {
 
 	containers, initContainers := p.toContainers(stats)
 	containerStatuses, initContainerStatuses := p.toContainerStatuses(stats)
-	starttime := metav1.NewTime(propertyTimestampToTime(p.m.ServiceProperty(name, "ExecMainStartTimestampMonotonic")))
```

Comment:
this requires a comment in the code, but i saw ExecMainStartTimestamp *not* being populated, while the monotonic did update... Which I should verify (somehow)
A bigger issue is that TimeStamp can go back in time because that probably system time and doesn't have to be in UTC?

Reply:
This is especially true if a unit has exited and we look at it's status after that (RemainAfterExit)

Reply:
https://unix.stackexchange.com/questions/234388/monitoring-systemd-service-activation-times

Reply:
What OS are you on?

This PR fixes the issue for me on RHEL 7.x, at least.

Reply:
Also fixes it for Ubuntu 20.04.1 as shown below.